lib/xml2.xml

```diff
@@ -22,15 +28,42 @@ such XML data files.</description>
     </implementation>
     <group>
       <requires interface="http://repo.roscidus.com/lib/zlib1g">
-      <environment insert="usr/lib" name="LD_LIBRARY_PATH"/>
-    </requires>
+        <environment insert="usr/lib" name="LD_LIBRARY_PATH"/>
+      </requires>
       <implementation id="sha1new=a7e5d754ed506f4e567fabeae05884d073a69433" released="2012-03-27" version="2.7.8">
-      <archive href="http://repo.roscidus.com/lib/xml2/libxml2-i386-2.7.8.dfsg-2+squeeze3.tar.bz2" size="809575"/>
-    </implementation>
+        <archive href="http://repo.roscidus.com/lib/xml2/libxml2-i386-2.7.8.dfsg-2+squeeze3.tar.bz2" size="809575"/>
+      </implementation>
       <implementation arch="Linux-x86_64" id="sha1new=9e9d62c9b0aec9ad55de90c0df16e08a8d29f373" released="2012-03-27" version="2.7.8">
-      <archive href="http://repo.roscidus.com/lib/xml2/libxml2-amd64-2.7.8.dfsg-2+squeeze3.tar.bz2" size="862772"/>
-    </implementation>
-    </group>
+        <archive href="http://repo.roscidus.com/lib/xml2/libxml2-amd64-2.7.8.dfsg-2+squeeze3.tar.bz2" size="862772"/>
+      </implementation>
+    </group>  
   </group>
-
-</interface>
\ No newline at end of file
+  <implementation arch="Windows-*" id="sha1new=865784de59ebfccf1b37d384fcfa7ec5c7b63dda" license="LGPL (GNU Lesser General Public License)" released="2001-12-23" version="2.4.12">
```

Comment:
Best to use `arch="Windows-i486"` for 32-bit DLLs.

Reply:
Done